app/assets/javascripts/Components/__tests__/submission_file_manager.test.jsx

```diff
@@ -2,7 +2,7 @@ import {SubmissionFileManager} from "../submission_file_manager";
 
 import {mount} from "enzyme";
 
-describe("For the submissions managed by SubmissionFileManager's FileManager child component", () => {
+describe("For the SubmissionFileManager", () => {
```

Comment:
Just for context, I had to refactor existing tests into a separate sub-suite so I could reuse the wrapper and the tests still made sense logically.